From 7f1e04d709037a6d3ab4f4692bc8c80e41b484bd Mon Sep 17 00:00:00 2001
From: Fabiano Francesconi <fabiano.francesconi@gmail.com>
Date: Wed, 3 Feb 2010 16:31:09 +0100
Subject: [PATCH] Fixed Pangoft2 check now using pkg-modules

---
 configure.in |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/configure.in b/configure.in
index c9b3ae0..6730123 100644
--- a/configure.in
+++ b/configure.in
@@ -7,6 +7,13 @@ AC_LANG_C
 AC_PROG_CC
 AM_PROG_LIBTOOL
 
+PANGOFT2_REQUIRED=1.26.0
+AC_SUBST(PANGOFT2_REQUIRED)
+
+PKG_CHECK_MODULES(pangoft2, pangoft2 >= $PANGOFT2_REQUIRED)
+AC_SUBST(pangoft2_CFLAGS)
+AC_SUBST(pangoft2_LIBS)
+
 DX_HTML_FEATURE(ON)
 DX_CHM_FEATURE(OFF)
 DX_CHI_FEATURE(OFF)
-- 
1.6.4.4

